src/playback/view/toolbars/components/timecodemodel.h

```diff
@@ -16,10 +16,10 @@ class TimecodeModel : public QAbstractListModel
 
     Q_PROPERTY(double value READ value WRITE setValue NOTIFY valueChanged FINAL)
 
-    Q_PROPERTY(double sampleRate READ sampleRate WRITE setSampleRate NOTIFY sampleRateChanged FINAL)
-    Q_PROPERTY(double tempo READ tempo WRITE setTempo NOTIFY tempoChanged FINAL)
-    Q_PROPERTY(int upperTimeSignature READ upperTimeSignature WRITE setUpperTimeSignature NOTIFY upperTimeSignatureChanged FINAL)
-    Q_PROPERTY(int lowerTimeSignature READ lowerTimeSignature WRITE setLowerTimeSignature NOTIFY lowerTimeSignatureChanged FINAL)
+    Q_PROPERTY(double sampleRate READ sampleRate WRITE setSampleRate FINAL)
```

Comment:
I reckon that these notifications aren't necessary and even confusing, since in fact this class won't change these values from the inside.
I was also wondering: is there a need for the `READ sampleRate` ? I tried without and it seemed to work (but since this seems harmless I left them).